compile-JM.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Installation directory
+export target=$(pwd)/install # this installs the binaries under install/bin/ in the present directory
+
+if [[ ! -d objdir ]]; then
+    mkdir objdir
+fi
+cd objdir
+
+ARMAFLAGS="-DARMA_NO_DEBUG -DARMA_DONT_USE_WRAPPER -llapack -lblas"
+export CXXFLAGS="-g -O2 -Wall -Wno-implicit-fallthrough -Wno-misleading-indentation ${ARMAFLAGS}"
+cmake ..  \
+          -DUSE_OPENMP=ON \
+          -DHELFEM_FIND_DEPS=ON \
+          -DCMAKE_INSTALL_PREFIX=${target} \
+          -DCMAKE_BUILD_TYPE=Release
+
+make -j4 VERBOSE=1 install
```

Comment:
No need for this file. Also, you appear to link to reference BLAS and LAPACK which is orders of magnitude slower than optimal versions. On Mac, I think the optimal library is [Accelerate](https://developer.apple.com/documentation/accelerate)

Reply:
`otool -L install/bin/gensap` confirms that the above (`-llapack -lblas`) has resulted in

```
../install/bin/gensap:
	/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib (compatibility version 1.0.0, current version 1.0.0)
	/System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libLAPACK.dylib (compatibility version 1.0.0, current version 1.0.0)
```

Reply:
The following `CMake.system`
```
# macOS with dependencies installed via Homebrew
add_definitions(-DHELFEM_FIND_DEPS=ON)
#
# link_libraries() is not a perfect option:
# 	https://stackoverflow.com/questions/17070101/why-i-cannot-link-the-mac-framework-file-with-cmake
# Unfortunately, 
#		target_link_libraries(helfem PRIVATE "-framework Accelerate")
# cannot be used here, as it must occur after add_executable() or add_library().
# Pragmatically, the following does successfully link against the wrappers from Apple's Accelerate framework
#		https://developer.apple.com/documentation/accelerate/blas
# if no other BLAS and LAPACK are installed:
link_libraries(blas lapack)
```
enables compilation on my good old MacBook Pro M1 using `compile.sh`.
Perhaps it could be useful to add as, e.g., `CMake.macOS_with_Homebrew`?

Reply:
You've added that file already, no? That one is OK. But this file duplicates `compile.sh` and is not necessary.